src/main/resources/db/migration/V0007__Insert_test_data_animal_images.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO `animal_images` VALUES (1,3,'testPath','jpeg',100500,'2023-08-23 08:39:35','2023-08-23 08:39:35'),
```

Comment:
What kind of data is this?

Reply:
It was first test only for table from DB

Reply:
Test ? If you write test use mock data or migration sql file only for test

Reply:
done